lib/Dialect/HLS/Transforms/ImplementTaskDesignSpace.cpp

```diff
@@ -154,8 +159,41 @@ struct ImplementTaskDesignSpacePattern : public OpRewritePattern<TaskOp> {
       }
       rewriter.eraseOp(linalgOp);
     } else {
+
+      //TRY
+      // auto symbol = implParamOp.getValue()->cast<TaskImplAttr>().getSymbolRef()
+      // auto default_space = SymbolTable::lookupNearestSymbolFrom<SpaceOp>(
+      //   op->getParentOfType<ModuleOp>(), symbol);
+
+      auto default_spaceOp = implSelect.getSpaces()[0].getDefiningOp<SpaceOp>();
```

Comment:
You should be able to directly use implSpaceOp here. Also, please follow LLVM naming conventions